src/server/routes/topics.ts

```diff
@@ -0,0 +1,32 @@
+import express from 'express'
+import type { Response, Request } from 'express'
+import { getTopics } from '../../utils/db'
+import { ObjectId } from 'mongodb'
+
+const topics = express.Router()
+
+topics
+  .get('/many', async (req: Request, res: Response) => {
+    const topicIds = req.body
+    const topics = getTopics()
+    const topic = await topics
+      .find({ _id: { $in: topicIds.map((id: string) => ObjectId(id)) } })
+      .toArray()
+    if (topic.length === 0) {
+      res.status(404).send(`Error: Topics not found.`)
+      return
+    }
+    res.send(topic)
+  })
+  .get('/:id', async (req: Request, res: Response) => {
```

Comment:
I really don't like how you're chaining these routes, I recommend `topics.get(...) ` for every route